pyaedt/icepak.py

```diff
@@ -412,8 +462,20 @@ def create_network_blocks(self, input_list, gravity_dir, top, assign_material=Tr
        
         Returns
         -------
-        type
-            Networks boundary objects.
+        list
+            List of boundary objects created.
+
+        Examples
+        --------
+
+        Create network boundaries from each box in the list. 
+
+        >>> box1 = icepak.modeler.primitives.create_box([1, 2, 3], [10, 10, 10], "NetworkBox2", "copper")
+        >>> box2 = icepak.modeler.primitives.create_box([4, 5, 6], [5, 5, 5], "NetworkBox3", "copper")
+        >>> input_list = [["NetworkBox2", 20, 10, 3], ["NetworkBox3", 4, 10, 2]]
+        >>> blocks = icepak.create_network_blocks(input_list, icepak.GravityDirection.ZNeg, 1.05918, False)
```

Comment:
@akaszynski it appears that doctest doesn't recognize multiline commands in the examples, so I just put the inputs in a list variable instead to shorten the line.

Reply:
@MaxJPRey informed me that you can use `...` for multiline commands, so I have updated this example with that.